index.js

```diff
@@ -115,17 +117,36 @@ function showScore() {
 //keycode: up 38, down 40, right 39, left 37
 // w: 87, a: 65, s: 83, d: 68
 function handleInput(e) {
-  if (e.keyCode === 39 || e.keyCode === 68) {
-    addNewDirection("right");
+  if (e.keyCode === 82) {
+    init();
+  }else if (e.keyCode === 39 || e.keyCode === 68) {
+    if(hardmode==0){
```

Comment:
It would be better if  you replace all double equals `==`  with triple equals `===`.